### PR TITLE
Docs: issue-3002: Uninstall using Ansible 

### PR DESCRIPTION
**Pull request by jenmalloy**
and related install and uninstall doc fixes

### CODE REVIEW

File: documentation/modules/proc-uninstalling.adoc [hunk 1]
```diff
@@ -18,7 +20,7 @@ ifeval::["{cmdcli}" == "oc"]
 endif::[]
 
 ifeval::["{cmdcli}" == "oc"]
-. Delete "cluster level" resources:
+. Delete the cluster-level resources:
```
Comment (jenmalloy) [inline documentation/modules/proc-uninstalling.adoc]: @k-wall, should this be "cluster-level custom resources"?

Comment (k-wall) [reply documentation/modules/proc-uninstalling.adoc]: @jenmalloy no, it is correct as it stands.

File: documentation/assemblies/assembly-installing.adoc [hunk 2]
```diff
@@ -28,12 +29,12 @@ endif::[]
 ifeval::["{cmdcli}" == "oc"]
 
 ifdef::OperatorSource[]
-{ProductName} can be installed by applying the YAML files using the `oc` command-line tool, by using
-the Operator Marketplace, or by running the link:https://www.ansible.com/[Ansible^] playbooks.
+{ProductName} can be installed by applying the YAML files using the OpenShift Container Platform command-line interface, by using
+the Operator Marketplace, or by running an link:https://www.ansible.com/[Ansible^] playbook.
```
Comment (k-wall) [inline documentation/assemblies/assembly-installing.adoc]: I think 'the' was correct, we ship one playbook.

**Comment by jenmalloy** [comment]
@k-wall, I changed "an Ansible playbook" to "the Ansible playbook" to address your comment, and left the other bit alone (per your comment)

**Pull request merged**